project/SbtUpdatesSettings.scala

```diff
@@ -10,6 +10,7 @@ object SbtUpdatesSettings {
     (Compile / compile) := ((Compile / compile) dependsOn dependencyUpdates).value,
     dependencyUpdatesFilter -= moduleFilter("org.scala-lang"),
     dependencyUpdatesFilter -= moduleFilter("com.typesafe.play"),
+    dependencyUpdatesFilter -= moduleFilter("uk.gov.hmrc", "essttp-backend-cor-journey"),
```

Comment:
we want to keep the BE version up to date in the stubs don't we?

Reply:
Yeah I guess, I just put it in to stop it failing incase there's a build race. I'll remove this and merge in backend in a bit. I need to address pr bot comments anyway